sys/net/npf/npf.h

```diff
@@ -59,6 +59,16 @@ typedef union {
 	uint32_t		word32[4];
 } npf_addr_t;
 
+struct r_uid {
```

Comment:
If you merge the two structs call it r_id { uint32_t id[2]; uint8_t op; };
Then you can merge most of the code.